hail_search/queries/base.py

```diff
@@ -296,7 +296,8 @@ def _parse_sample_data(self, sample_data):
     def _load_filtered_project_hts(self, project_samples, skip_all_missing=False, n_partitions=MAX_PARTITIONS, **kwargs):
         if len(project_samples) == 1:
             project_guid = list(project_samples.keys())[0]
-            project_ht = self._read_table(f'projects/{project_guid}.ht', use_ssd_dir=True)
+            sample_type = list(project_samples[project_guid].values())[0][0]['sample_type']
```

Comment:
This is a gross way to get the sample_type of the first sample from the first family, and it will change in the near future when we have multiple sample types in one family.